gnosis/safe-react-apps
# [Compound] Add COMP Claim support

## What it solves
Resolves #24

## How this PR fixes it
1) By substituting current APR calculation for the Supply and Distribution APYs showed in the Compound site
2) Adding a section for claiming COMP

## How to test it
1) Test APY calculations against Compound site. Should work both Mainnet and Rinkeby for the Supply APY and only in mainnet for distribution tokens (Distribution APY and COMP collect)
2) Try to collect available COMP. Any cToken should be supplied in mainnet for start collecting. We are showing the 18 decimals of the COMP token and allowing collection but once tested we should allow only to claim COMP greater than 0.0001 as in the Compound site

## Screenshots

![image](https://user-images.githubusercontent.com/1576776/147244891-3470872c-acae-4133-9430-17123ca52466.png)
